tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/appsettings.json

```diff
@@ -95,11 +95,11 @@
     "SubAreaOfLawDocumentTemplates": "DocumentTemplates",
     "ClientCustomPropertiesId": "ClientID",
     "ClientTermPath": "/",
-    "Level1Name": "pgTerms",
```

Comment:
Updated the appsettings.json to support generic taxonomy hierarchy -  will support upto 5 levels